### DISCUSSION TITLE
Issue when using helix editor

### DISCUSSION THREAD

**Discussion by Dangerise** [slint-ui/slint#5268] [Q&A]
I've created a discussions for helix, but no answer yet

Now I have a slint file, which define a component, then I rename the component, then I expect the new name of the component can be found in the auto completion of the editor

In vsc, it works well

but in helix, it doesn't work I expected

https://github.com/helix-editor/helix/assets/88434542/41b0c14d-b7cc-43ca-a649-65cd615ee90a

https://github.com/helix-editor/helix/assets/88434542/ce797644-36ea-467f-9e8e-ca9d7341c124

**Comment by hunger** [comment]
Interesting:-)

I *think* this is a rust-analyzer config issue... but I can not see a smoking gun in the [docs](https://github.com/rust-lang/rust-analyzer/blob/master/docs/user/generated_config.adoc) :-/
